packages/smooth_app/lib/query/product_query.dart

```diff
@@ -53,29 +53,64 @@ abstract class ProductQuery {
   }
 
   /// Returns the global country for API queries.
-  static OpenFoodFactsCountry? getCountry() => _country;
+  static OpenFoodFactsCountry getCountry() => _country;
+
+  /// Sets the global country for API queries: implicit choice at init time.
+  static Future<void> initCountry(
+    final UserPreferences userPreferences,
+  ) async {
+    const OpenFoodFactsCountry defaultCountry =
+        OpenFoodFactsCountry.SWITZERLAND; // why not?
```

Comment:
```suggestion
        OpenFoodFactsCountry.FRANCE; // not ideal, but we have many contributors monitoring France
```

Reply:
@teolemon OK for France. I haven't used your code suggestion because I wanted to avoid to reformat manually (your comment being too long).
I didn't work out, I have to reformat anyway...
We'll be ok in 5 minutes.